app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (
+    VaccineError,
+    NotWearingMaskError
+)
```

Comment:
You are using parentheses `()` for importing multiple classes which is good, but according to the checklist, each class should be on a separate line. Here is how it should look:

python
from app.errors import (
    VaccineError,
    NotWearingMaskError
)
```